web/src/enterprise/campaigns/global/marketing/CampaignsUserMarketingPage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { CampaignsMarketing } from './CampaignsMarketing'
+
+export interface CampaignsUserMarketingPageProps {
+    /** When true, the message shown will indicate that the user can just not use it because read-access has not been provided to non site-admins. */
+    enableReadAccess: boolean
+}
+
+export const CampaignsUserMarketingPage: React.FunctionComponent<CampaignsUserMarketingPageProps> = ({
+    enableReadAccess,
+}) => (
+    <CampaignsMarketing
+        body={
+            <section className="my-3">
+                <h2>Interested in Campaigns?</h2>
+                <p>
+                    At this time, creating and managing campaigns is only available to Sourcegraph admins. What can you
+                    do?
+                </p>
+                <div className="row">
+                    <ol>
+                        <li>Let your Sourcegraph admin know you're interested in using Campaigns for your team.</li>
+                        {enableReadAccess && (
+                            <li>
+                                Ask your Sourcegraph admin for{' '}
+                                <a href="https://docs.sourcegraph.com/user/campaigns#configuration" rel="noopener">
```

Comment:
```suggestion
                                <a href="https://docs.sourcegraph.com/user/campaigns" rel="noopener">
```

Everywhere else we link to the docs entry page, so I think it makes sense to be consistent